uploadNeuroDB/NeuroDB/objectBroker/InsertRole.pm

```diff
@@ -0,0 +1,90 @@
+package NeuroDB::objectBroker::InsertRole;
+
+=pod
+
+=head1 NAME
+
+NeuroDB::objectBroker::InsertRole -- A role for basic C<INSERT> operations on the database.
+
+=head1 DESCRIPTION
+
+This class provides a generic method to insert a given record in the database.
+If an operation cannot be executed successfully, a C<NeuroDB::objectBroker::ObjectBrokerException>
+will be thrown. All classes that use this role must implement methods C<getTableName()>, C<getColumnNames()> and C<db()>.
+
+=head2 Methods
+
+=cut
+
+use Moose::Role;
+
+requires 'getColumnNames';
+requires 'getTableName';
+requires 'db';
+
+use NeuroDB::DatabaseException;
+use NeuroDB::objectBroker::ObjectBrokerException;
+
+use TryCatch;
+
+=pod
+
+=head3 insertOne($valuesRef)
+
+Inserts the record with the properties passed as argument in the table whose name is returned
+by C<getTableName()>.
+
+INPUTS:
+   - reference to a hash array that contains the column values for the record to insert.
+     Each key must be a valid field (i.e that exsists in the array returned by C<getColumnNames()>
```

Comment:
typo in exists (one extra s ex**s**ists)

Reply:
Ok